uncaught.js

```diff
@@ -9,18 +11,15 @@ var jsonStringify = require('json-stringify-safe');
 
 var tryCatch = require('./lib/try-catch-it.js');
 var errors = require('./errors.js');
+var structures = require('./structures.js');
+var CONSTANTS = require('./constants.js');
 
-var LOGGER_TIMEOUT = 30 * 1000;
-var SHUTDOWN_TIMEOUT = 30 * 1000;
-var PRE_LOGGING_ERROR_STATE = 'pre.logging.error';
-var LOGGING_ERROR_STATE = 'logging.error';
-var PRE_GRACEFUL_SHUTDOWN_STATE = 'pre.graceful.shutdown';
-var GRACEFUL_SHUTDOWN_STATE = 'graceful.shutdown';
-var POST_GRACEFUL_SHUTDOWN_STATE = 'post.graceful.shutdown';
+var ALL_STATES = [];
```

Comment:
`ALL_STATES` contains all state machines in case multiple uncaught exceptions fire before we actually abort()

All the other states will contain the "in process" uncaught exceptions for the other uncaught exceptions.